specs/Standards/MultiToken/Events.md

```diff
@@ -0,0 +1,271 @@
+}
+
+// Transfer event log. Emitted when a token is transfered.  
+// Requirements
+// * Contract MUST emit event when transferring a token
+// Fields 
+// * `authorized_id`: approved account to transfer
+// * `old_owner_id`: the account sending the tokens "sender.near"
+// * `new_owner_id`: the account receiving the tokens "receiver.near"
+// * `token_ids`: the tokens to transfer 
+// * `amounts`: the number of tokens to transfer, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+//    array with 128 bits.
+interface MtTransferLog {
+    authorized_id?: string,
+    old_owner_id: string,
+    new_owner_id: string,
+    token_ids: string[],
+    amounts: string[],
+    memo?: string
+}
+// Approval event log. Emitted when a token's approval has changed.  
+// Requirements
+// * Contract MUST emit event when approval of tokens have changed
+// * Contract token_ids and amounts MUST be the same length 
+// Fields 
+// * `owner_id`: the account who owns the token, sending the approval
+// * `approved_account_id`: the account being approved
+// * `token_ids`: the tokens to transfer 
+// * `amounts`: the number of tokens approved for transfer, wrapped in quotes and treated
+//    like a string, although the numbers will be stored as an unsigned integer
+//    array with 128 bits.
+// * `memo`: optional message
+interface MtApprovalLog {
+    owner_id: string,
+    approved_account_id: string,
+    token_ids: string[],
+    amounts: string[],
+    memo?:  string
+}
+
+```
+
+## Examples
+
+Single owner minting (pretty-formatted for readability purposes):
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_mint",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts":["1", "100"]}
+  ]
+}
+```
+
+Different owners minting:
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_mint",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts":["1","100"]},
+    {"owner_id": "user1.near", "token_ids": ["meme"], "amounts": ["1"]}
+  ]
+}
+```
+
+Different events (separate log entries):
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_burn",
+  "data": [
+    {"owner_id": "foundation.near", "token_ids": ["aurora", "proximitylabs_ft"], "amounts": ["1","100"]},
+  ]
+}
+```
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_transfer",
+  "data": [
+    {"old_owner_id": "user1.near", "new_owner_id": "user2.near", "token_ids": ["meme"], "amounts":["1"], "memo": "have fun!"}
+  ]
+}
+```
+
+```js
+EVENT_JSON:{
+  "standard": "nep246",
+  "version": "1.0.0",
+  "event": "mt_approval",
+  "data": [
+    {"owner_id": "user1.near", "approved_account_id": "market.near", "token_ids": ["meme"], "amounts":["1"], "memo": "have fun at the market!"}
+  ]
+}
+```
+
```

Comment:
Let's please add one more example with `authorized_id`
See sad story here https://github.com/near/NEPs/pull/293

Reply:
Should correspond to the NFT examples now